app/questionPage.html

```diff
@@ -8,12 +8,27 @@
     <script src="js/dist/userLogIn.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
     <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
-    <script>
+    <script defer>
         let userEmail = getCookie('userEmail')
         getUser(userEmail).then(function (user) {
             if (!user.success || !isAuthorised(user.data) || user.data.canRetake === '0') {
                 window.location.replace("index.html")
             }
+            if (user.data.showTimer === '1'){
+                document.querySelectorAll('.timer').forEach(clock => {
+                    clock.hidden = false;
+                })
+                document.querySelectorAll('.hideTimerButton').forEach(hide_timer_button => {
+                    hide_timer_button.textContent = 'Hide Timer';
+                })
+            } else {
+                document.querySelectorAll('.timer').forEach(clock => {
+                    clock.hidden = true;
+                })
+                document.querySelectorAll('.hideTimerButton').forEach(hide_timer_button => {
+                    hide_timer_button.textContent = 'Show Timer';
+                })
+            }
             preventRetake(user)
         })
```

Comment:
it is not a good idea to add `defer` to an existing script tag. this changes the order of execution of the code and the script tag probably intentionally didn't have defer to start with.

you should create a separate script tag that has `defer` for this. some code duplication might be inevitable